src/Sulu/Bundle/CoreBundle/Resources/config/build.xml

```diff
@@ -21,14 +20,6 @@
             <tag name="massive_build.builder" />
         </service>
 
-        <service id="sulu_core.build.builder.cache" class="%sulu_core.build.builder.cache.class%">
-            <tag name="massive_build.builder" />
-        </service>
-
-        <service id="sulu_core.build.builder.fixtures" class="%sulu_core.build.builder.fixtures.class%">
```

Comment:
Why was this one removed?

Reply:
hmm i have to take a look currently no idea why this was removed ...

Reply:
will readd this